### PR TITLE
Added Social Links

### LINKED ISSUE #132: Add missing social links on top menu, sidebar and footer
Some social media links are missing. Please add them.

### PR DESCRIPTION
**Pull request by nzec**
* Added Font Awesome locally
* Added missing social links in Header,  Sidebar and footer in `404.html`, `500.html`, `index.html`.
* Fixes #132 .

**Here is the live link :** https://duttaditya18.github.io/2016.fossasia.org/index.html

If any image/script is not loading, then it is because of your AdBlock and not because I have made any changes.

Check it out @mariobehling @fossasia/web-developers @fossasia/website-admins

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -117,11 +117,12 @@
 
 							<div class="widget">
 								<ul class="social-profiles">
-									<li><a href="https://twitter.com/fossasia" target="default"><i class="icon social_twitter"></i></a></li>
-									<li><a href="https://facebook.com/fossasia" target="default"><i class="icon social_facebook"></i></a></li>
-									<li><a href="http://fossasia.org" target="default"><i class="icon social_dribbble"></i></a></li>
-									<li><a href="https://www.flickr.com/photos/fossasia/" target="default"><i class="icon social_instagram"></i></a></li>
-									<li><a href="https://plus.google.com/108920596016838318216" target="default"><i class="icon social_googleplus"></i></a></li>
+									<li><a target="default" href="http://twitter.com/fossasia"><i class="icon fa fa-twitter"></i></a></li>
```
Comment (abishekvashok) [inline index.html:120]: Please properly intend the code

**Review by abishekvashok — CHANGES_REQUESTED** [review]
Rest looks good :)

File: index.html (hunk 1 above)
Comment (nzec) [reply index.html:120]: @abishekvashok Fixed

**Comment by nzec** [comment]
@abishekvashok Fixed!

**Comment by duttaditya18** [comment]
@abishekvashok Can you review it now?

**Review by abishekvashok — APPROVED** [review]
ack https://github.com/fossasia/2016.fossasia.org/pull/133/commits/c0f29cb512a819eada69397dd7719514b1b8fd76

**Comment by nzec** [comment]
Thanks!

**Comment by duttaditya18** [comment]
Can anyone please review this please?

**Comment by anshumanv** [comment]
Any particular reason for adding font-awesome locally?

**Comment by nzec** [comment]
@anshumanv Yes. See this : https://github.com/fossasia/2014.fossasia.org/issues/26
mariobehling recommends it.

**Review by anshumanv — APPROVED** [review]

**Comment by ParthS007** [comment]
@duttaditya18  Why there is space on both sides ?
image

**Review by ParthS007 — APPROVED** [review]

**Pull request merged**